tests/foreman/cli/test_user.py

```diff
@@ -247,6 +253,50 @@ def test_positive_create_user_6(self):
             self.fail(err)
         self.__assert_exists(args)
 
+    @skip_if_bug_open('bugzilla', 1213426)
+    def test_create_user_with_default_location(self):
+        """@Test: Check if user with default location can be created
+
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new default location assigned
+
+        @BZ: 1213426
+
+        """
+        try:
+            location = make_location()
+            user = make_user(
+                {'location-ids': location['id'],
+                 'default-location-id': location['id']},
+            )
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.assertIn(location['name'], user['locations'])
+        self.assertEqual(location['name'], user['default-location'])
+
+    @skip_if_bug_open('bugzilla', 1213426)
+    def test_create_user_with_defaut_org(self):
+        """@Test: Check if user with default organization can be created
+
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new default organization assigned
+
+        @BZ: 1213426
+
+        """
+        try:
+            org = make_org()
+            user = make_user(
```

Comment:
please format dicts with more than one element like the following:

```python
user = make_user({
    'a': 'b',
    'c': 'd',
})
```

Be aware of the trailing `,` on the last line, it is required in our style convention.